### PR TITLE
CRM-20875 - Fix membership import to avoid notice error to be thrown

### PR DESCRIPTION
**Pull request by jitendrapurohit**
* [CRM-20875: Import of membership custom data throws notice errors.](https://issues.civicrm.org/jira/browse/CRM-20875)

### CODE REVIEW

**Comment by seamuslee001** [comment]
This looks sensible to me @eileenmcnaughton does it make sense to you? @jitendrapurohit is there anyway to get a test added to this?

File: CRM/Member/Import/Parser/Membership.php [hunk 1]
```diff
@@ -279,7 +279,8 @@ public function import($onDuplicate, &$values) {
       $session = CRM_Core_Session::singleton();
       $dateType = $session->get('dateTypes');
       $formatted = array();
-      $customFields = CRM_Core_BAO_CustomField::getFields(CRM_Utils_Array::value('contact_type', $params));
+      $customDataType = !empty($params['contact_type']) ? $params['contact_type'] : 'Membership';
+      $customFields = CRM_Core_BAO_CustomField::getFields($customDataType);
```
Comment (monishdeb) [inline CRM/Member/Import/Parser/Membership.php]: onto single line change by 
```
 $customFields = CRM_Core_BAO_CustomField::getFields(CRM_Utils_Array::value('contact_type', $params, 'Membership'));
```
If ```$customDataType``` is not used below

**Comment by monishdeb** [comment]
I think in case of notice error or any user warning we can avoid adding UT unless it's affecting the workflow.  Still it's always good to have test coverage wherever it's possible

**Comment by eileenmcnaughton** [comment]
I agree that @monishdeb change to one line is nicer - but I think it is minimal enough I don't think this needs to be held on that.

Re unit tests - once again I agree that they are preferable. However, I also acknowledge that this is a very small change & I'm going to merge it

**Comment by jitendrapurohit** [comment]
@monishdeb Apologies for not pushing the one-liner for the change. I thought of testing it on php7 and then update,  but somehow missed.

**Pull request merged**